scheduler/src/cook/mesos.clj

```diff
@@ -52,18 +52,18 @@
      (let [submit-time (java.util.Date.)]
        (doseq [{:keys [uuid command ncpus memory name retry-count priority]} jobs
                :let [txn {:db/id (d/tempid :db.part/user)
-                          :job/uuid uuid
```

Comment:
Since none of these fields are optional, my naive interpretation is they are all equally important, in that they must have a value.
I can see wanting to place some fields closer to each other (e.g. cpu and mem), but I went with sorting as it makes searching for a field much easier and introduces an unwritten protocol for insertion locations.

Does this perspective help change your preference?